presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5003,6 +5003,19 @@ public void testDistributedExplainGraphvizFormat()
     }
 
     @Test
+    public void testExplainValidate()
+    {
+        MaterializedResult result = computeActual("EXPLAIN (TYPE VALIDATE) SELECT 1");
+        assertEquals(result.getOnlyValue(), true);
+    }
+
+    @Test(expectedExceptions = Exception.class)
```

Comment:
I would use `expectedExceptionsMessageRegExp` or a more specific exception type. Otherwise we might get a false negative, if it failed with the wrong sort of exception